src/desktop/apps/auction/queries/sale.js

```diff
@@ -2,59 +2,77 @@ export default function SaleQuery(id) {
   return `
   query SaleQuery {
     sale(id: "${id}") {
-      _id
-      associated_sale {
-        ${require("../../../components/react/auction_block/query.js").default}
+      internalID
+      associated_sale: associatedSale {
+        cover_image: coverImage {
+          cropped(width: 260, height: 110) {
+            url
+          }
+        }
+        end_at: endAt
+        href
+        id: slug
+        is_closed: isClosed
+        is_live_open: isLiveOpen
+        is_preview: isPreview
+        live_start_at: liveStartAt
+        name
+        start_at: startAt
       }
-      auction_state
-      cover_image {
-        cropped(width: 1800 height: 600 version: "wide") {
+      status
+      cover_image: coverImage {
+        cropped(width: 1800, height: 600, version: "wide") {
           url
         }
       }
       currency
       description
-      eligible_sale_artworks_count
-      end_at
+      eligible_sale_artworks_count: eligibleSaleArtworksCount
+      end_at: endAt
       id
-      is_auction
-      is_closed
-      is_live_open
-      is_open
-      live_start_at
+      is_auction: isAuction
+      is_closed: isClosed
+      is_live_open: isLiveOpen
+      is_open: isOpen
+      live_start_at: liveStartAt
       name
-      promoted_sale {
+      promoted_sale: promotedSale {
         id
         name
-        sale_artworks {
-          artwork {
-            id
-            title
-            date
-            sale_message
-            is_in_auction
-            image {
-              placeholder
-              url
-              aspect_ratio
-            }
-            artists {
-              id
-              href
-              name
-            }
-            partner {
-              name
+        saleArtworksConnection(first: 99) {
```

Comment:
I'm sorry I only seem to be able to come up with this type of comment, but: does this mean only the first page of lots is fetched?